dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/net/UploadStatus.kt

```diff
@@ -16,43 +16,48 @@ internal enum class UploadStatus {
     HTTP_REDIRECTION,
     HTTP_CLIENT_ERROR,
```

Comment:
I would propose to add a description to each value saying it retry will be made or not. Adding `_RETRY` suffix made me think that other statuses won't retry, which is not true.